yarn.lock

```diff
@@ -13173,6 +13173,13 @@ supports-color@^7.0.0, supports-color@^7.1.0:
   dependencies:
     has-flag "^4.0.0"
 
+supports-color@^7.1.0:
```

Comment:
? no package.json change, this seems out of place. merge conflict resolution error?

Reply:
Uh oh, for sure. It should be: `supports-color@^7.0.0, supports-color@^7.1.0`